src/cca/app/scripts/cordova_browser_api.ts

```diff
@@ -343,6 +347,14 @@ class CordovaBrowserApi implements BrowserAPI {
     }
   }
 
+  public exit = () => {
+    (<any>navigator).Backbutton.goBack(() => { /* MT (success) */ }, () => {
```

Comment:
Yep, looks good.  Technically the separate d.ts files are per-plugin.  `navigator.app` seems to be a bit of a special case.  It's defined by Cordova itself, so arguably it should go in cordova.d.ts, but the code is actually in a special built-in plugin called "app":
https://github.com/apache/cordova-android/blob/457c5b8b3b694265c991b456b15015741ade5014/cordova-js-src/plugin/android/app.js

So your current version is fine by me.